pkg/apis/ops/v1alpha1/register.go

```diff
@@ -20,10 +20,18 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-var SchemeGroupVersion = schema.GroupVersion{
-	Group:   "ops.antrea.tanzu.vmware.com",
-	Version: "v1alpha1",
-}
+// GroupName is the group name used in this package.
+const GroupName = "ops.antrea.tanzu.vmware.com"
+
+// VersionNumber is the version number used in this package.
+const VersionNumber = "v1alpha1"
+
+var (
```

Comment:
Unneeded parentheses.
BTW, why we have these two constants above, I guess we can retrieve them by using SchemeGroupVersion?

Reply:
Reset.